tests/virtual_machines_test.go

```diff
@@ -518,19 +518,35 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 
 			Context("and kubemacpool is opted-in on a namespace", func() {
+				var (
+					vm *kubevirtv1.VirtualMachine
+				)
 				BeforeEach(func() {
-					By("opting out the namespace")
+					By("opting in the namespace")
 					err := addLabelsToNamespace(TestNamespace, map[string]string{vmNamespaceOptInLabel: "allocate"})
```

Comment:
nit: Checking `Succeed` is nicer.

Reply:
the comment is not in the correct place (I guess github is to blame) but I guess you meant this line:
```
Expect(err).ToNot(HaveOccurred(), "should success creating the vm")
```

fixed. Done

Reply:
No idea how it landed on this line indeed. But it was meant for all occurrences where the return value is only an error, which there are a few of those at the moment.

Then you can do `Expect(foo()).To(Succeed())`

Reply:
ack. this is true for all the tests, will do in a separate PR